package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "6.0.0",
```

Comment:
I've been keeping this version in my PRs here for the UIs. My thinking was that the jump to `6.0.0` would have everything legal name related. So this way, when we merge with main, main would be `6.0.0` with the legal name update.

Thoughts?

Reply:
It feels like it might be nice to deploy new code as 6.0.0 but that's not what the industry typically does -- the first "clean" version of something might be something like 6.3.10.

Meanwhile, I think it's nice to update the minor or patch version so we can always tell what version of the UI we're looking at.

Reply:
Got it. Makes sense. Thanks for clarifying Sev.